### ISSUE TITLE
Unable to Accept license when exporting docker image on Windows

### ISSUE THREAD

**Issue by russellseymour** [habitat-sh/habitat#6566]
**OS**: Windows 10
**Docker Version**: 2.0.0.3 (31259)
**Habitat Versions**:
_Local_: 0.79.1
_In Export_: 0.81.0

I am trying to export a nuilt HART file as a docker image on Windows. I have configured Docker for Windows to work with Windows containers.

If I try to export the image *without* setting the `HAB_FEAT_INSTALL_HOOK` then the image is created successfully. However none of the WIndows components, such as IIS and Static content are installed.

If I set the `HAB_FEAT_INSTALL_HOOK` to 1 then I can see that all the Windows components are installed as required. However when it comes to performing the export it fails because Habitat 0.81.0 requires the license to be accepted.

I have tried to set the `HAB_LICENSE` environment variable and this does not get put into the command that runs the Habitat supervisor.

![image](https://user-images.githubusercontent.com/791658/57837102-fd124980-77b9-11e9-823e-51e6233b30c7.png)

The command that runs in the export correctly has the feature hook but not the license variables:

![image](https://user-images.githubusercontent.com/791658/57837176-29c66100-77ba-11e9-801a-e5b6d7f45fb2.png)

After looking in this repo I came across a PR that I thought might help (#6142) so I tried to run the export in a Docker studio (I had previously been running direct from the command line) but this does not work either. In fact there is a new error:

![image](https://user-images.githubusercontent.com/791658/57837313-77db6480-77ba-11e9-8ff2-4c34d64c52ae.png)

As can be seen the Docker Server OS does not appear to be set to anything, but Docker is set to run Windows containers.

In an attempt to work around this I set a `pkg_dep` in my `plan.ps1` for `core/hab/0.79.1` in the hope that this would come through to the Docker export. However it still pulls in version 0.81.0.

I have tried to use Habitat 0.81.0 on the machine, and this time I can see that the `HAB_LICENSE` is being added to the command within the export. However it still fails.

![image](https://user-images.githubusercontent.com/791658/57838042-e53bc500-77bb-11e9-9d30-c8943ddfaad7.png)

I understand that there is a regression bug for Windows exports which is causing this issue. The advice has been to use 0.79.1 but as can be seen from the screen shots above this does not affect the version of Habotat (0.81.0) pulled into the export.

So in summary it is not possible to accept the license agreement for a Windows export on a Windows machine.

Is there any work around for this?
 - Downgrade Habitat to 0.79.1 in the export
 - Force the HAB_LICENSE into the export command

**Comment by russellseymour** [comment]
I have been able to get an export working by adding a couple of arguments to the export command. These are to pin the version of Habitat that is installed in the Docker container.

So my command now looks like:

```
hab pkg export docker results\russellseymour-movieapp_com-master-20190516081701-x86_64-windows.hart --sup-pkg core/hab-sup/0.79.1 --hab-pkg core/hab/0.79.1
```

As can be seen from the screen shot below, the correct version of Hab is being used.

![image](https://user-images.githubusercontent.com/791658/57861971-7aa47c80-77ef-11e9-9c76-2ea12684f3a5.png)

Thanks very much to @christophermaier

**Comment by mwrock** [comment]
Thanks for posting this workaround @russellseymour ! I am going to close this since it is fixed by #6540
